docs/turn-howto.md

```diff
@@ -42,65 +38,112 @@ This will install and start a systemd service called `coturn`.
-    for this purpose.
+    > You may need to install `libevent2`: if so, you should do so in
+    > the way recommended by your operating system. You can ignore
+    > warnings about lack of database support: a database is unnecessary
+    > for this purpose.
 
 1.  Build and install it:
 
         make
         make install
 
+#### Docker installation
+There are coturn docker images available. You can prepare by running
+
+        docker pull instrumentisto/coturn
+
+The first part of the command needed to start your container detached is as follows:
+
+        docker run -d -p 3478:3478/udp -p 3478:3478/tcp \
+        -p <range_start-range_end:range_start-range_end>/udp \
+        --name="<fill in container name for easy access in CLI>" \
+        -v /etc/localtime:/etc/localtime:ro \
+        --mount type=tmpfs,destination=/var/lib/coturn \
+        --restart=unless-stopped instrumentisto/coturn
+
+With the first -p option we expose the TURN port on both TCP and UDP. The second
+-p option specifies the UDP port range used for media bridging (where your clients
+will send media streams). The temp mount is to store run state of the deamon.
+See Docker documentation for details.
+
+For Docker all coturn configuration options listed below can also be specified
+on the docker startup script, like this:
+
+        [... same as above to start ...]
+        --restart=unless-stopped instrumentisto/coturn \
+        --total-quota=1200 --realm=<your turn server domain, ie turn.domain.tld> \
+        --external-ip='$(detect-external-ip)'
+
+So any configuration option described below you can add after the docker image
+name, preceeded by --. You can also mount a volume with -v and place a
+configuration file there, for example turnserver.conf.
+
+
 ### Configuration
 
 1.  Create or edit the config file in `/etc/turnserver.conf`. The relevant
     lines, with example values, are:
 
         use-auth-secret
-        static-auth-secret=[your secret key here]
+        static-auth-secret=<your secret key here>
         realm=turn.myserver.org
+        external-ip='$(detect-external-ip)'
+        no-cli
+
+    The 'external-ip' can either be the included detection script like this example
+    or the real external IP. That IP can be different from the local / relay IP, which
+    is auto detected and not specified.
 
     See `turnserver.conf` for explanations of the options. One way to generate
     the `static-auth-secret` is with `pwgen`:
 
         pwgen -s 64 1
 
-    A `realm` must be specified, but its value is somewhat arbitrary. (It is
-    sent to clients as part of the authentication flow.) It is conventional to
-    set it to be your server name.
-
-1.  You will most likely want to configure coturn to write logs somewhere. The
-    easiest way is normally to send them to the syslog:
+2.  You will most likely want to configure coturn to write logs somewhere. The easiest way is normally to send them to the syslog:
```

Comment:
please don't undo the existing wrapping. (Also, why manually renumber the list items?)